src/register-default-commands.coffee

```diff
@@ -31,11 +32,15 @@ module.exports = ({commandRegistry, commandInstaller, config, notificationManage
     'application:unhide-all-applications': -> ipcRenderer.send('command', 'application:unhide-all-applications')
     'application:new-window': -> ipcRenderer.send('command', 'application:new-window')
     'application:new-file': -> ipcRenderer.send('command', 'application:new-file')
-    'application:open': -> ipcRenderer.send('open-command', 'application:open')
-    'application:open-file': ->
+    'application:open': -> 
+      defaultPath = atom.workspace.getActiveTextEditor()?.getPath() ? atom.project.getPaths()?[0]
+      ipcRenderer.send('open-command', 'application:open', defaultPath)
+    'application:open-file': -> 
       defaultPath = atom.workspace.getActiveTextEditor()?.getPath() ? atom.project.getPaths()?[0]
       ipcRenderer.send('open-command', 'application:open-file', defaultPath)
-    'application:open-folder': -> ipcRenderer.send('open-command', 'application:open-folder')
+    'application:open-folder': -> 
+      defaultPath = atom.workspace.getActiveTextEditor()?.getPath() ? atom.project.getPaths()?[0]
```

Comment:
This bothers me, having `defaultPath` defined three places. But `atom` is only available when these functions are called later, so it has to be defined inside the callback. If you know of a better way to write this, let me know.